notebooks/MAST/HSC/CASJOBS/HCV_API/HCV_API_demo.ipynb

```diff
@@ -0,0 +1,1039 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Description of the variable classification columns <a name=\"classification\"></a>\n",
+    "\n",
+    "Several of the table columns have information on the variability.\n",
+    "\n",
+    "* The columns `AutoClass` and `ExpertClass` have summary information on the variability for a given `MatchID` object.\n",
+    "    * `AutoClass`: Classification as provided by the system: 0=constant 1=single filter variable candidate (SFVC) 2=multi-filter variable candidate (MFVC)\n",
+    "    * `ExpertClass`: Classification as provided by expert: 0=not classified by expert, 1=high confidence variable, 2=probable variable, 4=possible artifact\n",
+    "* The columns `MAD` and `Chi2` are variability indices using the median absolute deviation and the $\\chi^2$ parameter for the given filter.\n",
+    "* The column `VarQualFlag` is a variability quality flag (see Section 5 of the paper). The five letters correspond to CI, D, MagerrAper2, MagAper2-MagAuto, p2p; AAAAA corresponds to the highest quality flag.\n",
+    "* The column `FilterDetFlag` is the filter detection flag: 1=source is variable in this filter, 0=source is not variable in this filter.\n",
+    "\n",
+    "See the HCV paper by [Bonanos et al. (2019, AAp)](https://www.aanda.org/component/article?access=doi&doi=10.1051/0004-6361/201936026) for more details on the computation and meaning of these quantities."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Find objects with measurements in both F475W and F814W\n",
+    "\n",
+    "This could be done in a SQL query through the CasJobs interface.  Here we use the `Astropy.table.join` function instead."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "w475 = np.where(tab['Filter'] == 'ACS_F475W')\n",
+    "w814 = np.where(tab['Filter'] == 'ACS_F814W')\n",
+    "\n",
+    "# the only key needed to do the join is MatchID, but we include other common columns\n",
+    "# so that join includes only one copy of them\n",
+    "jtab = join(tab[w475], tab[w814],\n",
+    "            keys=['MatchID', 'GroupID', 'SubGroupID', 'RA', 'Dec', 'AutoClass', 'ExpertClass'],\n",
+    "            table_names=['f475', 'f814'])\n",
+    "print(len(jtab), \"matched F475W+F814W objects\")\n",
+    "jtab[jtab['AutoClass'] > 0]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Plot object positions on the sky <a name=\"positions\"></a>\n",
+    "\n",
+    "We mark the galaxy center as well.  Note that this field is in the outskirts of IC 1613.  The 0.5 degree search radius (which is the maximum allowed in the API) allows finding these objects."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plt.rcParams.update({'font.size': 16})\n",
```

Comment:
instead of setting the rcParams in every plot, I think it makes more sense to put it all at the very top.  After all, then the user can fiddle with it based on their screen size, etc.